### ISSUE TITLE
Build a web server in assembly

### ISSUE THREAD

**Issue by ACHREF999** [codecrafters-io/build-your-own-x#1272]
### Main programming language

Assembly

### Tutorial title

Web in Assembly

### Tutorial URL
https://www.youtube.com/playlist?list=PLpM-Dvs8t0VbLCQEKNhIIIHdTPlGVvETz

### Category
* [ ] 3D Renderer
* [ ] Augmented Reality
* [ ] BitTorrent Client
* [ ] Blockchain / Cryptocurrency
* [ ] Bot
* [ ] Command-Line Tool
* [ ] Database
* [ ] Docker
* [ ] Emulator / Virtual Machine
* [ ] Front-end Framework / Library
* [ ] Game
* [ ] Git
* [ ] Network Stack
* [ ] Neural Network
* [ ] Operating System
* [ ] Physics Engine
* [ ] Programming Language
* [ ] Regex Engine
* [ ] Search Engine
* [ ] Shell
* [ ] Template Engine
* [ ] Visual Recognition System
* [ ] Voxel Engine
* [ ] Web Search Engine
* [ x ] Web Server
* [ ] Uncategorized

**Comment by SitaGanesh** [comment]
The "Web Server" category doesn't have a checkbox like the others. Should this tutorial be added there, or is there a different category or action you're looking for? Let me know how you'd like to proceed.

**Comment by ACHREF999** [comment]
I honestly didnt know how to add a checkbox 🤣
Sorry for the incinvenience

**Comment by SitaGanesh** [comment]
Yeah that's completely fine! But was in search of issues to tackle, so I just saw and asked randomly like that the 'checkbox' is an issue aa?😅. 
Okay I am new to contribution and I have hands on practice of CLI of git. The main thing is i struggle in getting into nice organisation, and after getting into it what would be my role one thing. What happens if I make any mistake. This is killing me and making me stop. 
So can u just explain the process if possible, how do the organisation work, how should I get into it, what I need to do first. How do I understand there codebase and everything thats all.

**Comment by ACHREF999** [comment]
idk what to say , good lcuk i guess
[image: image.png]
